source/utils/generate-stack.ts

```diff
@@ -0,0 +1,10 @@
+/**
+Generates a useful stacktrace for your verification.
```

Comment:
"for your verification" What does this mean?

Reply:
That message really needs a better wording, I agree. What that basically does is generate a more accurate stack trace to where the code actually errored. How does ``Generates a useful stacktrace that points to the user's code where the error happened, on platforms without the `Error.captureStackTrace` function present`` sound like?